modules/@apostrophecms/import-export-page/index.js

```diff
@@ -30,10 +33,17 @@ module.exports = {
       post: {
         import: [
           require('connect-multiparty')(),
```

Comment:
not related to your PR but we should put require statement at the top

Reply:
required at the top of the file